ui/src/containers/Drawer/components/Slide.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import styles from '../DrawerStyle.scss'
+
+export default function Slide({ itemList }: { itemList: string[] }) {
+  return (
+    <div className={styles.slide}>
+      <div className={styles.slide_inner}>
+        {itemList.map((member, idx) => (
+          <div key={idx} className={styles.slide_item}>
```

Comment:
avoid using index as component key directly. suggest to give something more meaningful:
```suggestion
          <div key={`slideItem${idx}`} className={styles.slide_item}>
```

Reply:
fixed in 1111810